src/ibek/runtime_cmds/commands.py

```diff
@@ -140,3 +141,55 @@ def generate_index(title: str, index_entries: List[IndexEntry]):
 
     """
     DLSFormatter().format_index(title, index_entries, GLOBALS.OPI_OUTPUT / "index.bob")
+
+
+@runtime_cli.command()
+def generate_autosave(subst_file: Path = typer.Argument(GLOBALS.RUNTIME_SUBSTITUTION)):
+    """
+    Generate autosave request files from autosave settings and positions
```

Comment:
@coretl please could you have a read of this comment and decide if you like what I have done here.

Reply:
Looks fine